guillotina/contrib/dbusers/__init__.py

```diff
@@ -0,0 +1,31 @@
+from guillotina import configure
+from guillotina.i18n import MessageFactory
+
+_ = MessageFactory("guillotina.contrib.dbusers")
+
+
+app_settings = {
+    "auth_user_identifiers": [
+        "guillotina.contrib.dbusers.users.DBUserIdentifier"
+    ]
+}
+
+configure.permission("guillotina.NotAuthenticated", "")
+configure.permission("guillotina.Authenticated", "")
+configure.permission("guillotina.AddUser", title="Add plone user")
+configure.permission("guillotina.AddGroup", title="Add plone group")
+configure.grant(permission="guillotina.AccessContent", role="guillotina.Anonymous")
+configure.grant(permission="guillotina.NotAuthenticated", role="guillotina.Anonymous")
+configure.grant(permission="guillotina.Authenticated", role="guillotina.Authenticated")
+configure.grant(permission="guillotina.AddUser", role="guillotina.ContainerAdmin")
+configure.grant(permission="guillotina.AddGroup", role="guillotina.ContainerAdmin")
+
```

Comment:
Only principals with `guillotina.ContainerAdmin` role will be able to add new groups/users.

At the same time, the `guillotina.ManageUsers` permission is only given to `guillotina.Manager`

is that what we want? Shouldn't at least `guillotina.Manager` have `guillotina.AddUser` and `guillotina.AddGroup` permission?

Reply:
Yes, probably!

Reply:
from the role description, I don't see why `ContainerAdmin` should have permissions to manage/create users.

Unless someone have reasons to do so, for now I gave those to `guillotina.Manager`

Reply:
I think it makes sense for `ContainerAdmin` to have this access. `guillotina.Manager` is a very powerful user that usually only the root user has.

ContainerAdmin is more specific to managing containers or a specific container only.